core/neat/mixins/_grid-media.scss

```diff
@@ -1,8 +1,15 @@
 @charset "UTF-8";
-/// Creates a media query in which custom grid properties can be defined.
-/// Add the `media` property to your custom grid settings map and add the media
-/// you would like to scope to. If only a number is defined, it is assumed this
-/// is a `min-with` value. Your custom grid can then be passed to the mixin.
+/// Neat supports complex changes to the grid applied by Neat's other mixins
```

Comment:
I'm also having a bit of a hard time parsing this.

 - `grid-media` allows you to scope custom properties to a custom media query. 
 - Invoking a custom `grid-media` allows you to temporarily apply neat grid properties, such as custom gutters or columns to that specific query only.